### YamlMime:PythonClass
uid: azure.batch.custom.custom_errors.CreateTasksErrorException
name: CreateTasksErrorException
fullName: azure.batch.custom.custom_errors.CreateTasksErrorException
module: azure.batch.custom.custom_errors
inheritances:
- builtins.Exception
summary: Aggregate Exception containing details for any failures from a task add operation.
constructor:
  syntax: CreateTasksErrorException(pending_tasks=None, failure_tasks=None, errors=None)
  parameters:
  - name: message
    description: Error message describing exit reason
    types:
    - '[str](https://docs.python.org/3.6/library/stdtypes.html#str)'
  - name: pending_tasks
    description: List of tasks remaining to be submitted.
    types:
    - '[<xref:azure.batch.models.TaskAddParameter>]'
  - name: failure_tasks
    description: List of tasks which failed to add
    types:
    - '[<xref:azure.batch.models.TaskAddResult>]'
  - name: errors
    description: List of unknown errors forcing early termination
    types:
    - '[[~Exception](https://docs.python.org/3.6/library/exceptions.html#Exception)]'
